solutionbox/code_free_ml/test_mltoolbox/test_analyze_data.py

```diff
@@ -715,7 +716,7 @@ def _open_and_encode_image(img_url):
         img = Image.open(f).convert('RGB')
         output = cStringIO.StringIO()
         img.save(output, 'jpeg')
-      return output.getvalue()
+      return base64.urlsafe_b64encode(output.getvalue())
```

Comment:
so the tft image transform expects this column to be base64, but there is no decode op in analyze_data.py

Where is this decoded to regular bytes?

I think analyze_data and transform_raw_data assume the input tensor is a regular byte string. But the code in the test files show the exported graph assumes the column is a base64 string.

Said again, I thin analyze_data and transform_raw_data need updating to start from a base64 string. This might be wasteful. Another option is to have a custom serving input function that applies tf.decode_base64